plugins/circleci-deploy/src/index.ts

```diff
@@ -28,6 +29,31 @@ const generateConfigWithServerlessOptions = (
   return generateConfigWithJob(jobOptions)
 }
 
+const getServerlessAdditionalFields = (logger: Logger): JobConfig => {
```

Comment:
**nitpick**: could we get the other classes to use this new function instead of `generateConfigWithServerlessOptions` to reduce repetition? or have `generateConfigWithServerlessOptions` call this function?